src/C4Config.h

```diff
@@ -156,6 +156,7 @@ class C4ConfigSound
 	int32_t MusicVolume;
 	int32_t SoundVolume;
 	std::chrono::seconds SoundCommandCooldown;
+	bool MuteSoundChatCommand; // whether to mute /sound by default
```

Comment:
The existing configuration value uses `SoundCommand` instead of `SoundChatCommand` - this should be consistent.